restore-cognito.md

```diff
@@ -0,0 +1,32 @@
+Prerequisites 
+For creating and executing backups, you can use the npm package called `cognito-backup-restore` 
+(do `npm install cognito-backup-restore` to get the package)
+
+Required IAM permissions
+If you want to create a backup of the users you will need the permission - `ListUsers`
+Restoring the users requires - `AdminCreateUser`
+There might be more roles required to do all the operations for getting and restoring users. If you need a permission added
+speak with Iva or Pol, who will add it through IAM.
+
+**Downloading user data**
+Here do either 1 or 2 depending on the situation.
+
+1. In case there is no cognito user pool or the user pool is empty, we have weekly user backup which executes every sunday at 1am. 
+To find the latest users backup file you need to go to biomage-backups-production S3 bucket and find the latest object in the bucket. 
+There you can download the `data.json` file, which later will be used to import the users to the new/existing user pool.
+
+2. In case the users are still existing in a pool you can create a new backup file from the user pool, which can then be used to import users.
```

Comment:
Similarly to the comment for point 1, I will shorten the description of point 2 to say only `2. In case the users are still existing in a pool` and then move the rest of the sentence as a new sentence in a new line.